core/stat/base/metric_bucket.go

```diff
@@ -76,8 +76,16 @@ func (mb *MetricBucket) reset() {
 func (mb *MetricBucket) AddRt(rt int64) {
 	mb.addCount(base.MetricEventRt, rt)
 	if rt < atomic.LoadInt64(&mb.minRt) {
-		// Might not be accurate here.
-		atomic.StoreInt64(&mb.minRt, rt)
+		// Use CompareAndSwapInt64 to store the new min value atomically.
```

Comment:
Does it have to be so precise? It has to be measured between performance and accurate，I think it's better left untouched

Reply:
ok,thank you!